Atomic/atomic.py

```diff
@@ -379,6 +379,14 @@ def set_args(self, args):
         except:
             pass
 
+        try:
+            self.storage = args.storage
+        except:
+            self.storage = None
+
+        if not self.storage:
+            self.storage = self.get_atomic_config_item(["default_storage"])
+
         if not self.name and self.image is not None:
```

Comment:
What happens is self.storage is still None?  Will everything still work. or will the system throw a  proper exception?